pkg/solana/chain.go

```diff
@@ -238,11 +238,14 @@ func newChain(id string, cfg *config.TOMLConfig, ks loop.Keystore, lggr logger.L
 		var nodes []mn.Node[mn.StringID, *client.Client]
 
 		for i, nodeInfo := range cfg.ListNodes() {
+			if nodeInfo == nil || nodeInfo.Name == nil || nodeInfo.URL == nil {
+				return nil, fmt.Errorf("node config contains nil: %+v", nodeInfo)
+			}
```

Comment:
this might be useful here?
https://github.com/smartcontractkit/chainlink-solana/blob/a7050b5193cde253b9f649c2bd59b083a34636a3/pkg/solana/config/config.go#L153

Reply:
Nice, will add this to config validation instead!